mission_control/navigator_launch/launch/simulation.launch

```diff
@@ -38,6 +38,11 @@
         <param name="autorepeat_rate" value="5"/>
     </node>
 
+    <node pkg="navigator_emergency_control" type="navigator_emergency_control" name="navigator_emergency_xbee">
```

Comment:
Any teleop control in the sim is disabled in favor of perfect tracking for mission testing. 
The best way to test funcationality is to look at `wrench/cmd` and make sure that is doing what you want it to.
There's some changes that could be made to the sim to let teleop be added back in if someone wants to do that as well.